src/poliastro/maneuver.py

```diff
@@ -234,7 +245,11 @@ def bielliptic(cls, orbit_i, r_b, r_f):
         t_trans1 = np.pi * np.sqrt(a_trans1 ** 3 / k)
         t_trans2 = np.pi * np.sqrt(a_trans2 ** 3 / k)
 
-        return cls((0 * u.s, dv_a), (t_trans1, dv_b), (t_trans2, dv_c))
+        return cls(
+            (tof.decompose(), dv_a.decompose()),
+            (t_trans1.decompose(), dv_b.decompose()),
```

Comment:
Same as above, please confirm we don't have to add `tof` (which shouldn't in fact be called `tof`, but `t_pericenter` or something like that? to avoid confusion) to the other times.